frontend/src/css/Sidebar.styl

```diff
@@ -71,9 +78,10 @@ borderWhenActive= 5px
       & > span
         display flex
         align-items center
+        padding-inline-start standardSpacing
     &__space
       display flex
-      padding-inline-start borderWhenActive
+      padding-inline-start spacingInsideMenu + 10px
```

Comment:
Might be _nothing_ but shouldn't we have a defined variable for space padding?

Reply:
Fixed